recipe/meta.yaml

```diff
@@ -1,14 +1,16 @@
-{% set version = "3.0.0b3" %}
-{% set version_str = "v3.0.0-beta-3" %}
+{% set name = "protobuf" %}
+{% set version = "3.0.0" %}
+{% set version_str = "v3.0.0" %}
+{% set sha256 = "f5b3563f118f1d3d6e001705fa7082e8fc3bda50038ac3dff787650795734146" %}
 
 package:
-    name: protobuf
+    name: {{ name }}
     version: {{ version }}
 
 source:
-    fn: protobuf_{{ version }}.tar.gz
-    url: https://github.com/google/protobuf/archive/{{ version_str }}/protobuf-{{ version_str }}.tar.gz
-    md5: 89afd3855f2d4782e59c09e07d9efa67
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://github.com/google/protobuf/archive/{{ version_str }}/{{ name }}-{{ version_str }}.tar.gz
+    sha256: {{ sha256 }}
 
 build:
     number: 0
```

Comment:
Could we please rebase/merge with `master` and then set this to `0`? Please see this [line]( https://github.com/conda-forge/protobuf-feedstock/blob/91c9b2c304a98bd4221815ae4649f14ff7ed8124/recipe/meta.yaml#L14 ) in `master`. So this will be `1` when we merge right now even though the diff doesn't seem to show it.